bingads_downloader/config.py

```diff
@@ -8,11 +8,19 @@ def data_dir() -> str:
     return '/tmp/bingads/'
 
 
-def data_file() -> str:
-    """The name of the file the result is written to"""
+def ad_performance_data_file() -> str:
+    """The name of the file the ad performance result is written to"""
     return 'ad_performance.csv.gz'
 
 
+def keyword_performance_data_file() -> str:
+    """The name of the file the keyword performance result is written to"""
+    return 'keyword_performance.csv.gz'
+
+def campaign_performance_data_file() -> str:
+    """The name of the file the campaign performance result is written to"""
+    return 'campaign_performance.csv.gz'
```

Comment:
Campaign performance is not mentioned in the readme or CLI. Is there a reason?

Reply:
No, I forgot to remove it after removing that logic. Removed now